src/cosmos/upgrade/v1beta1/query.ts

```diff
@@ -56,7 +55,29 @@ export interface QueryUpgradedConsensusStateRequest {
  * RPC method.
  */
 export interface QueryUpgradedConsensusStateResponse {
-  upgradedConsensusState?: Any;
+  upgradedConsensusState: Uint8Array;
```

Comment:
This seems to be a breaking change

Reply:
Good catch. It seams however that this is not used - nobody is calling `SetUpgradedConsensusState`, so there is no response.

Reply:
It was changed here: https://github.com/cosmos/cosmos-sdk/pull/8673/files#r682168269